docs/about/organization.rst

```diff
@@ -1,3 +1,5 @@
+:html_theme.sidebar_secondary.remove:
+
 Organization
```

Comment:
In general - there is a mix of English and British spellings in the docs. I suggest we keep consistent. In the above we have American, which i personally prefer. Let's stick with one form.

Reply:
I agree, we should stick to one.